oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/BackChannelLogoutRequestHandler.java

```diff
@@ -102,7 +109,7 @@ public JWTClaimsSet generateClaims(BackChannelLogoutMessage inputEvent) {
                 .audience(inputEvent.getClientId())
                 .subject(inputEvent.getSubjectId())
                 .expirationTime(clock.nowPlus(2, ChronoUnit.MINUTES))
-                .issuer(instance.getOidcApiBaseURL().orElseThrow())
+                .issuer(oidcApi.baseURI().toString())
```

Comment:
Have we tested in dev that these produce exactly the same result wrt things like trailing slashes? I'd imagine there are RPs doing exact string matching on this and any change would be breaking

Reply:
Will be easier to test in the tokens we generate that are shown in the stub, and that should be sufficient

Reply:
Should be the same. `.baseURI` effectively calls `.getOidcApiBaseURL` and the URI class respects the inclusion / exclusion or trailing slashes as you'd expect.